Neos.Neos/Classes/Aspects/FusionCachingAspect.php

```diff
@@ -46,4 +58,22 @@ public function cacheGetMergedFusionObjectTree(JoinPointInterface $joinPoint)
 
         return $fusionObjectTree;
     }
+
+    /**
+     * Get a site for the given site node.
+     *
+     * See same method in the FusionService
+     *
+     * @param TraversableNodeInterface $siteNode
+     * @return Site
+     */
+    protected function getSiteForSiteNode(TraversableNodeInterface $siteNode): Site
+    {
+        /** @noinspection PhpUndefinedMethodInspection */
+        $site = $this->siteRepository->findOneByNodeName((string)$siteNode->getNodeName());
+        if ($site === null) {
+            throw new \RuntimeException(sprintf('Failed to fetch site for node "%s"', (string)$siteNode->getNodeName()), 1606132610);
```

Comment:
Sure but now this isn't in line with the original in the FusionService anymore, which would fail hard anyways.

Reply:
Ah right..  Doh

Reply:
OK, I'll revert this and remove the corresponding test

Reply:
Thank you <3